package.json

```diff
@@ -55,6 +55,12 @@
       "react-scripts>eslint-plugin-testing-library": ">=5",
       "react-scripts>@typescript-eslint/parser": ">=5",
       "react-scripts>@typescript-eslint/eslint-plugin": ">=5"
+    },
+    "peerDependencyRules": {
+      "allowedVersions": {
+        "cypress": "*",
+        "jest": "*"
+      }
```

Comment:
pnpm enables [`strict-peer-dependencies`](https://pnpm.io/npmrc#strict-peer-dependencies) by default, now, so I was getting some errors due to `cypress-image-snapshots`. This is the only way I managed to solve them.